### DISCUSSION TITLE
Entrega de prácticas

### DISCUSSION THREAD

**Discussion by GloriaCoralCerecedo** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#58] [Q&A] [answered]
Entonces las practicas por mencionar un ejemplo se podrían entregar en vez del 25 el 28. Pero el MC puso esa fecha para no atrasarnos.

**Comment by Fullgamer494** [comment, marked as answer]
Así es, queda en ti entregarlos conforme al tiempo "ideal" o el que ellos tienen previsto para poder abarcar todos los contenidos correctamente, o ir a tu ritmo

**Reply by GloriaCoralCerecedo** [reply]
Gracias
